vpdq/cpp/pdq/cpp/hashing/pdqhashing.cpp

```diff
@@ -2,8 +2,6 @@
 // Copyright (c) Meta Platforms, Inc. and affiliates.
 // ================================================================
 
-#include <mutex>
```

Comment:
Any ideas for in the long term how we can ensure pdq and vpdq say in sync? Should we write a test that checks that file contents of the vpdq directory match the cpp one?